tsconfig.cli.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Why do we need 2 tsconfig files?

Reply:
Let's experiment with it in a separate PR. Maybe it's a good idea to move /cli to /src. Build artifacts should live separately.